merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/json/SerializedValueSerializer.java

```diff
@@ -0,0 +1,113 @@
+package gov.nasa.jpl.aerie.merlin.protocol.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public final class SerializedValueSerializer extends StdSerializer<SerializedValue> {
+
+  public SerializedValueSerializer() {
+    super((Class<SerializedValue>) null);
```

Comment:
Instead of the `null` cast I'd just call `this(null)` here instead.

Reply:
It looks like that's needed because the super constructor is overloaded.

Reply:
Yup! But when using `this(null)` the other constructor that you added will be hit.
```java
  public SerializedValueSerializer(Class<SerializedValue> c) {
    super(c);
  }
```

Reply:
Using `this()` in this context will basically perform an implicit cast.

Reply:
TIL github doesn't have a facepalm emoji :)

Reply:
done. thanks!

Reply:
All good! It's subtle!